plugins/colorbutton/plugin.js

```diff
@@ -331,31 +359,47 @@ CKEDITOR.plugins.add( 'colorbutton', {
 					colorLabel = colorName;
 				}
 
-				output.push( '<td>' +
-					'<a class="cke_colorbox" _cke_focus=1 hidefocus=true' +
-						' title="', colorLabel, '"' +
-						' draggable="false"' +
-						' ondragstart="return false;"' + // Draggable attribute is buggy on Firefox.
-						' onclick="CKEDITOR.tools.callFunction(', clickFn, ',\'', colorCode, '\',\'', type, '\'); return false;"' +
-						' href="javascript:void(\'', colorCode, '\')"' +
-						' data-value="' + colorCode + '"' +
-						' role="option" aria-posinset="', ( i + 2 ), '" aria-setsize="', total, '">' +
-						'<span class="cke_colorbox" style="background-color:#', colorCode, '"></span>' +
-					'</a>' +
-					'</td>' );
+				output.push( '<td>' + generateTileHtml( {
```

Comment:
As I wrote in comment, `CKEDITOR.dom.element.createFromHtml()` can't be used to create `<td>` element. So if I encapsulate also `<td>` element, I will not be able to use this function to create color boxes (I think that's a good name, and it's already present in codebase) in other places 😢. If we could fix `CKEDITOR.dom.element.createFromHtml()` before, it would be obvious to do what you proposed.